service/authorizer/openapi.json

```diff
@@ -73,6 +73,17 @@
       },
       "type": "object"
     },
+    "authorizerv2Decision": {
```

Comment:
@gimmyxd, notice that the latest openapi `buf` plugin renamed `v2Decision` to `authorizerv2Decision`. I assume that's in order to disambiguate from the `api.Decision` (in decision_logs.proto).

Reply:
There should be no issue, we will regenerated typescript bindings and update the console.